js/directives/lucca/user-picker.js

```diff
@@ -586,14 +596,56 @@
 			}
 		};
 
+		/*************************/
+		/***** DISPLAY USERS *****/
+		/*************************/
+
+		var displaySelectedUserFirst = function(users) {
+			var selectedUser = _.find(users, function(user) { return user.id === selectedUserId; });
+
+			// Remove 'selected' and 'dividing' class from all users
+			_.each(users, function(user) {
+				if (user.isSelected) {
+					user.isSelected = false;
+				}
+			});
+
+			if (selectedUser) {
+				// The class 'dividing' and 'selected' will be applied to the user
+				selectedUser.isSelected = true;
+			}
+			return sortUsers(selectedUserId, users);
+		};
+
+		// Display the user with the given id as first result
+		var sortUsers = function(id, users) {
+			var sortedUsers = users;
+			var userIds = _.pluck(users, "id");
+
+			if (_.contains(userIds, id)) {
+				var partitions = _.partition(users, function(user) { return (user.id === id); }); // [[user], [rest]]
+				// Sort users with 'user' as first result
+				sortedUsers = _.union(partitions[0], partitions[1]);
+			}
+			return sortedUsers;
+		};
+
 		/*********************/
 		/***** ON-SELECT *****/
 		/*********************/
 
 		$scope.updateSelectedUser = function(selectedUser) {
 			$scope.onSelect();
-			// Bind the selected user to the ng-model in luid-user-picker directive
-			$scope.ngModel = selectedUser;
+
+			selectedUserId = selectedUser.id;
```

Comment:
i dont like that, if your ng-model evolves by another method than the updateSelectedUser, then `selectedUserId` is deprecated and you will highlight the wrong user

you should add the require to the ngModel in the directive definition and a method to the `luidUserPickerCtrl` during the `link` function to get the id of the `$viewValue`

```js
upCtrl.getSelectedUser = function(){
  return ngModelCtrl.$viewValue;
}
```